draft-ietf-httpbis-priority.md

```diff
@@ -634,6 +634,32 @@ signal the intended priority to the client by including the Priority field in
 a PUSH_PROMISE or HEADERS frame.
 
 
+# Retransmission Scheduling
+
+A transport may detect packet losses and then retranmit lost information to
+provide reliability. Examples include TCP and QUIC. While this document
+specifies HTTP layer prioritization, its effectiveness can be further achieved
+if transport layer also respect such priorities. Besides scheduling new data,
+transport implementations MAY also consider stream priorities during
+retransmission. In this section we will use QUIC as an example to discusss a
+couple scenarios an implementation need to consider.
+
+{{!I-D.ietf-quic-transport}}, Section 13.3 states that endpoints SHOULD 
+prioritize retransmission over sending new data, unless otherwise specified by
+application. When an HTTP/3 application uses priority scheme defined in this 
+document, the QUIC transport layer MAY take stream priorities as input to 
+scheduling the sending of both retransmission data and new data. For example, 
+if one stream with higher urgency has new data to send and another stream with 
+lower urgency has retransmission data to send, a QUIC transport may choose to 
+schedule the higher urgency new data before the lower urgency retransmission
+data.
+
+{{?I-D.ietf-quic-recovery}}, Section 6.2.4 also highlights consideration of 
+application priorities when sending probe packets after PTO timer expiration.
+An implementation using this prioritization scheme for new or retransmitted
+data, MAY choose a different strategy for prioritizing probe data. 
```

Comment:
```suggestion
{{?I-D.ietf-quic-recovery}}, Section 6.2.4 also highlights consideration of
application priorities when sending probe packets after PTO timer expiration. An
QUIC implementation supporting application-indicated priorities might use the
relative priority of streams when choosing probe data. 
```

I think it is clearer to just make this a consideration point that implementers should be aware of, so removing normative language.